addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComBlinds1Message.java

```diff
@@ -269,7 +269,7 @@ public void convertFromState(RFXComValueSelector valueSelector, Type type) throw
                     command = Commands.STOP;
 
                 } else {
-                    throw new NumberFormatException("Can't convert " + type + " to Command");
+                    throw new RFXComException("Can't convert " + type + " to Command");
```

Comment:
Sorry I just spotted it and had no logical other branch in which it fitted, if needed I can revert it.